kpops/components/streams_bootstrap_v2/base.py

```diff
@@ -140,3 +143,9 @@ def warning_for_latest_image_tag(self) -> Self:
                 f"The image tag for component '{self.name}' is set or defaulted to 'latest'. Please, consider providing a stable image tag."
             )
         return self
+
+    @pydantic.model_validator(mode="before")
+    @classmethod
+    def deprecation_warning(cls, model: Any) -> Any:
+        log.warning("StreamsBootstrapV2 is deprecated, use StreamsBootstrap instead.")
```

Comment:
rephrased a little, since it comes with breaking changes
```suggestion
        log.warning("StreamsBootstrapV2 is deprecated, consider migrating to StreamsBootstrap.")
```